packages/pact-testing/templates/run-with-mock-bff.sh

```
@@ -0,0 +1,77 @@
+# Calculate package root (two levels up from pact/scripts)
+PACKAGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+
+# ========================================
+# LOAD SHARED UTILITIES
+# ========================================
+# Source shared shell helper functions from pact-testing package
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+HELPERS_FILE="$SCRIPT_DIR/../../../pact-testing/src/helpers/shell-helpers.sh"
+
+if [[ -f "$HELPERS_FILE" ]]; then
+    source "$HELPERS_FILE"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)